app/pb_kits/playbook/pb_bar_graph/_bar_graph.jsx

```diff
@@ -21,13 +21,16 @@ type BarGraphProps = {
   title: String,
   type?: String,
   legend?: Boolean,
+  legendClick?: Boolean,
   height?: String,
 }
 
 export default class BarGraph extends React.Component<BarGraphProps> {
   static defaultProps = {
     className: 'pb_bar_graph',
     type: 'column',
+    legend: false,
+    legendClick: true,
```

Comment:
@jasoncypret should we instead keep the existing functionality and make `legendClick: false` by default?

Reply:
There was a discussion in the playbook connect chat the other day about this. When setting a prop that will change default behaviour, it's dangerous to existing functionality within Nitro. If we make this `false` by default, I'm going to have to submit a PR changing _everywhere_ it's being used in Nitro. That or we leave it as an opt-in.

The reason I'm concerned about this is because the functionality for `legend` was set to default to `false` last week, and nitro was not touched. This has caused a customer facing page to lose the legend, making the graph confusing to the customer.

Thoughts?

Reply:
Thanks for the quick reply! Ok, based on this info, that doesn't sound good then. I had this backwards. LGTM